src_code/src/controllers/services/Inventory/GetAvailableModels.js

```diff
@@ -0,0 +1,102 @@
+/** Initialize neccessary modules */
+const responseBuilder = require("../../../utils/interfaces/IResponseBuilder");
+const db = require("../../../configurations/database/DatabaseConfigurations");
+const dbHelpers = require("../../../utils/interfaces/IDBHelperFunctions");
+
+/**
+ * Retrieve available models within a specified date range.
+ * @param {Object} res - Express response object.
+ * @param {string} startDate - Start date for availability check.
+ * @param {string} endDate - End date for availability check.
+ * @returns {Object} - Response object indicating success or failure.
+ */
+async function GetAvailableModels(res, startDate, endDate) {
```

Comment:
Um, how do you pass in startDate and endDate? 😕

Reply:
It is in the comments where we defined the routes. 
/**
 * GET/RETRIEVE ALL AVAILABLE EQUIPMENTS/MODELS IN A PERIOD OF TIME
 * URL => /api/inventory/available-models?startDate=value&endDate=value
 * Response Body: 
 * {
 *  "message": "Available models successfully retrieved.",
 *  "responseObject": 
 *   [
 *      {
 *          "modelId": 59,
 *          "modelName": "Oakton Aneroid",
 *          "modelPhoto": "https://drive.google.com/uc?id=1NvjXSvJtM1RJQbVAQIqs9NhY4ptF0oUT",
 *          "typeName": "Barometer",
 *          "availableCount": 2
 *      }
 *   ]
 * }
 */